### PR TITLE
updating stress paths to match feature

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/stress/scripts/azure_eventhub_producer_stress.py [hunk 1]
```diff
@@ -169,27 +171,39 @@ def create_client(self, client_class, is_async=False):
...
+            client = client_class(
+                fully_qualified_namespace=self.args.hostname,
                 eventhub_name=self.args.eventhub,
+                credential=DefaultAzureCredential(),
                 auth_timeout=self.args.auth_timeout,
                 http_proxy=http_proxy,
                 transport_type=transport_type,
                 logging_enable=self.args.uamqp_logging_enable,
-                **retry_options
+                **retry_options
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/stress/scripts/azure_eventhub_producer_stress.py:184]: nit probably doesn't matter:
```suggestion
                **retry_options
```

**Review by swathipil — APPROVED** [review]
lgtm! (left one tiny nit that you can just ignore)

File: sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml [hunk 2]
```diff
@@ -10,8 +10,51 @@ spec:
...
+      command: ['bash', '-c', 'python azure_eventhub_consumer_stress_async.py --azure_identity True --duration 7200']
+      {{- end -}}
+
+      {{ if eq .Stress.Scenario "consumesyncidentity" }}
+      command: ['bash', '-c', 'python azure_eventhub_consumer_stress_sync.py --azure_identity True --duration 7200']
+      {{- end -}}
+
       {{- include "stress-test-addons.container-env" . | nindent 6 }}
-      # async test command
-      # command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py -m stress_send_list_async & python3 azure_eventhub_consumer_stress_async.py']
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml:16]: do we plan to test async too? or is that a future plan?

Comment (l0lawrence) [reply sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml:16]: the added test scenarios cover async and sync (i.e sendlistconsumeasync), these lines are removing the old commented out code. I do notice line 31 is sync it should be async -- will fix that!

**Pull request merged**